src/core/mesh/qgsmeshlayer.h

```diff
@@ -142,46 +142,9 @@ class CORE_EXPORT QgsMeshLayer : public QgsMapLayer
     QgsTriangularMesh *triangularMesh() SIP_SKIP;
 
     //! Returns renderer settings
-    QgsMeshRendererMeshSettings rendererNativeMeshSettings() const;
-
-    //! Sets new renderer settings, triggers repaint
-    void setRendererNativeMeshSettings( const QgsMeshRendererMeshSettings &settings );
-
-    //! Returns renderer settings
-    QgsMeshRendererMeshSettings rendererTriangularMeshSettings() const;
-
-    //! Sets new renderer settings, triggers repaint
-    void setRendererTriangularMeshSettings( const QgsMeshRendererMeshSettings &settings );
-
-    //! Returns renderer settings
-    QgsMeshRendererScalarSettings rendererScalarSettings() const;
-
-    //! Sets new renderer settings, triggers repaint
-    void setRendererScalarSettings( const QgsMeshRendererScalarSettings &settings );
-
-    //! Returns renderer settings
-    QgsMeshRendererVectorSettings rendererVectorSettings() const;
-
-    //! Sets new renderer settings, triggers repaint
-    void setRendererVectorSettings( const QgsMeshRendererVectorSettings &settings );
-
-    /**
-     * Sets active scalar dataset for rendering
-     *
-     * Triggers repaint
-     */
-    void setActiveScalarDataset( QgsMeshDatasetIndex index = QgsMeshDatasetIndex() );
-    //! Returns active scalar dataset
-    QgsMeshDatasetIndex activeScalarDataset() const { return mActiveScalarDataset; }
-
-    /**
-     * Sets active vector dataset for rendering.
-     *
-     * If dataset is not vector based, do nothing. Triggers repaint
-     */
-    void setActiveVectorDataset( QgsMeshDatasetIndex index = QgsMeshDatasetIndex() );
-    //! Returns active vector dataset
-    QgsMeshDatasetIndex activeVectorDataset() const { return mActiveVectorDataset; }
+    QgsMeshRendererSettings rendererSettings() const;
+    //! Sets new renderer settings
```

Comment:
I guess this should be `\since QGIS 3.4`? Is this somewhere tagged as unstable API?

Reply:
The whole class is tagged as experimental in the doxygen docs of the class. The idea is to make the mesh layer API stable from 3.4 so I did not bother to mark new/changed functions...